src/mbgl/map/transform.hpp

```diff
@@ -144,7 +144,12 @@ class Transform : private util::noncopyable {
     bool isPanning() const { return state.isPanning(); }
 
     // Conversion and projection
-    ScreenCoordinate latLngToScreenCoordinate(const LatLng&) const;
+    enum class ConversionMode : bool {
+        FindShortestPath = 0,
+        IgnoreShortestPath
```

Comment:
👍 `cameraForLatLngs()` itself may end up taking a `ConversionMode` option as part of the fix for #8708.